tox.ini

```diff
@@ -69,6 +69,7 @@ download = true
 deps =
     coveralls: coveralls
     esmpy: git+https://github.com/esmf-org/esmf.git@v{env:ESMF_VERSION}\#subdirectory=src/addon/esmpy
+    xclim: git+https://github.com/Ouranosinc/xclim.git@main
```

Comment:
is this still necessary with the new release ?

Reply:
I figured we could leave it there and have at least one build that checks against the main branch. For xclim, we have a dedicated upstream build; Would it be prudent to implement that here as well? (in another PR)

Reply:
Ah I see. no strong opinions from me for the upstream build.

Reply:
You say "no strong opinions", I read "no opposition, go for it". You got it. ;)